File: Python/program40.py
def Check_Prime(iNo):
    
    if(iNo < 0):
        iNo = -iNo 
    
    bFlag = True
    for iCnt in range(2,(iNo // 2) + 1):
        if((iNo % iCnt) == 0):
            bFlag = False
            break
        
    return bFlag

           
def main():
    
    iValue = int(input("Enter number\n"))
    
    bRet = Check_Prime(iValue)
    
    if(bRet == True):
        print(iValue, " is prime number\n")
    else:
        print(iValue, " is not prime number\n")

if __name__ == "__main__":
    main()

File: C++/program40.cpp
#include<iostream>
using namespace std;

int CountEvenDigit(int iNo)
{
    int iDigit = 0, iCount = 0;

    while(iNo != 0)
    {
        iDigit = iNo % 10;
        if((iDigit % 2) == 0)
        {
            iCount++;
        }
        iNo = iNo / 10;
    }

    return iCount;

}

int main()
{
    int iValue= 0, iRet = 0;

    cout<<"Enter number"<<'\n';
    cin>>iValue;

    iRet = CountEvenDigit(iValue);

    cout<<"Number of even digits are "<<iRet<<'\n';

    return 0;
}